extensions/wikia/ExactTargetUpdates/tests/adapters/WikiCategoriesAdapterTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+class WikiCategoriesAdapterTest extends WikiaBaseTest {
+
+	public function setUp() {
+		$this->setupFile = __DIR__ . '/../../ExactTargetUpdates.setup.php';
+		parent::setUp();
+	}
+
+	public function testGetCategoriesMapping() {
+		// Params
+		$param = new stdClass();
+		$param->Properties->Property = [
+			$this->buildResultPair( 'city_id', 1 ),
+			$this->buildResultPair( 'cat_id', 3 )
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "cat_id" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=php%3AS1192)